launchable/utils/link.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+from typing import Dict, List, Mapping
+
+JENKINS_URL_KEY = 'JENKINS_URL'
+JENKINS_BUILD_URL_KEY = 'BUILD_URL'
+GITHUB_ACTIONS_KEY = 'GITHUB_ACTIONS'
+GITHUB_ACTIONS_SERVER_URL_KEY = 'GITHUB_SERVER_URL'
+GITHUB_ACTIONS_REPOSITORY_KEY = 'GITHUB_REPOSITORY'
+GITHUB_ACTIONS_RUN_ID_KEY = 'GITHUB_RUN_ID'
+GITHUB_PULL_REQUEST_URL_KEY = "GITHUB_PULL_REQUEST_URL"
+CIRCLECI_KEY = 'CIRCLECI'
+CIRCLECI_BUILD_URL_KEY = 'CIRCLE_BUILD_URL'
```

Comment:
[question]
CIRCLECI_BUILD_URL_KEY = 'CIRCLECI_BUILD_URL' ?

Reply:
`CIRCLE_BUILD_URL` is correct 
![image](https://user-images.githubusercontent.com/536667/213080655-fc5fbb15-5c23-43c7-bcb5-a04a49b246ea.png)

ref: https://circleci.com/docs/variables/

Reply:
ok